### PR TITLE
No accept refuse

### PR DESCRIPTION
**Pull request by camilleldn**

### CODE REVIEW

**Comment by camilleldn** [comment]
screen shot 2016-01-19 at 9 45 42 am

**Comment by camilleldn** [comment]
Fixes #14

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -49,3 +49,7 @@
   font-weight: 700;
   font-size: 24px;
 }
+
+.width-action-select {
+  width: 59%;
```
Comment (camilleldn) [inline app/assets/stylesheets/application.css.scss]: #forreasons ;)

File: app/controllers/occupations_controller.rb [hunk 2]
```diff
@@ -12,11 +12,14 @@ def update
 
   private
 
-  def occupation_accepted?(params)
-    params[:commit].downcase != 'refuse'
+  def occupation_accepted?(_params)
+    not_no_actions = action_params[:actions].select do |action|
```
Comment (gjvis) [inline app/controllers/occupations_controller.rb]: `action_params[:actions].any? { |action| -> action != 'no') }`

File: app/models/action.rb [hunk 3]
```diff
@@ -1,26 +1,40 @@
...
+    {
+      key: 'no',
+      title: 'This role isn\'t interesting to me at the moment'
+    },
+    {
+      key: 'notes',
+      title: 'My notes'
     }
-  }.freeze
+  ].freeze
```
Comment (gjvis) [inline app/models/action.rb]: Why not use a Hash?

File: app/controllers/occupations_controller.rb [hunk 4]
```diff
@@ -12,8 +12,8 @@ def update
 
   private
 
-  def occupation_accepted?(params)
-    params[:commit].downcase != 'refuse'
+  def occupation_accepted?(_params)
+    action_params.fetch(:actions, []).any? { |action| action != 'no' }
```
Comment (gjvis) [inline app/controllers/occupations_controller.rb]: for added fun and magic, a bit of array coercion:
```
Array(action_params[:actions]).any? { |action| action != 'no }
```

not saying its better, just that its fun :tada: 

(fetch is probably clearer)

**Comment by camilleldn** [comment]
Fixed #15 and #18

**Comment by gjvis** [comment]
:ship: :ship: :ship: :ship: :ship: :ship: :ship: v

**Pull request merged**